packages/core/src/components/date-input/date-input.tsx

```diff
@@ -0,0 +1,458 @@
+          onInput={(event) => this.onFromInputChange(event)}
+          required
+        />
+        <span class="vertical-align">
+          <ix-icon name="arrow-right"></ix-icon>
+        </span>
+        <input
+          id="secondInput"
+          ref={(ref) => (this.secondInput = ref as HTMLInputElement)}
+          type="text"
+          class="form-control"
+          placeholder={this.format}
+          onFocus={this.onInputFocus}
+          onBlur={this.onInputBlur}
+          value={this._to}
+          onInput={(event) => this.onToInputChange(event)}
+          required
+        />
+      </Fragment>
+    );
+  }
+
+  renderSingleInput(): any {
+    return (
+      <Fragment>
+        <input
+          ref={(ref) => (this.firstInput = ref as HTMLInputElement)}
+          id="firstInput"
+          type="text"
+          class="form-control"
+          placeholder={this.format}
+          onFocus={this.onInputFocus}
+          onBlur={this.onInputBlur}
+          value={this._from}
+          onInput={(event) => this.onFromInputChange(event)}
+        />
+      </Fragment>
+    );
+  }
+
+  renderLabel(isInsideInput: boolean): any {
+    if (
+      !this.label ||
+      (isInsideInput && !(this.labelPosition === 'inside')) ||
+      (!isInsideInput && this.labelPosition === 'inside')
+    ) {
+      return '';
+    }
+
+    return (
+      <span
+        id="date-input-label"
+        class={{
+          'vertical-align':
+            this.labelPosition === 'left' || this.labelPosition === 'inside',
+          'left-position': this.labelPosition === 'left',
+          'inside-position': this.labelPosition === 'inside',
+        }}
+      >
+        {this.label}
+      </span>
+    );
+  }
+
+  render() {
+    return (
+      <Host class={{ 'label-flex': this.labelPosition === 'left' }}>
+        {/* renders if position is top/left */}
+        {this.renderLabel(false)}
+        <div
+          class="date-input"
+          aria-labelledby="date-input-label"
+          ref={(ref) => (this.dateInputDiv = ref)}
+        >
+          {/* renders if position is inside */}
+          {this.renderLabel(true)}
+          {this.range ? this.renderRangeInput() : this.renderSingleInput()}
+          <span
+            class={{
+              'icon-button': true,
+              clear: true,
+            }}
+          >
+            <ix-icon-button
+              ghost
+              icon="clear"
+              class={{ hidden: !this._from && !this._to }}
+              onClick={this.clear}
+            ></ix-icon-button>
+          </span>
+          <span class="icon-button">
+            <ix-icon-button ghost icon="chevron-down-small"></ix-icon-button>
+          </span>
+        </div>
+        <ix-dropdown
+          trigger={this.dateInputDiv}
+          closeBehavior="outside"
+          onClick={(event) => event.stopPropagation()}
+          class="dropdown"
+        >
+          <ix-date-picker-rework
+            tabIndex={0}
```

Comment:
The picker shows the default white border on focus. Ideally the focus would jump to an interactive element directly.

Reply:
https://github.com/siemens/ix/pull/861/commits/e98290a4da66a99f43fa1d92ee9796abc6bc7f13